packages/jest-runtime/src/script_transformer.js

```diff
@@ -217,7 +217,6 @@ export default class ScriptTransformer {
     if (transform && shouldCallTransform) {
       const processed = transform.process(content, filename, this._config, {
         instrument,
-        returnSourceString: false,
```

Comment:
We need this option to not have a breaking change on our hands

Reply:
Okay. But in the future it is going to be removed right? After giving proper deprecation messages?

Reply:
Yeah, we should remove it in the next major of babel-jest :)